_pages/en/capabilities.md

```diff
@@ -71,7 +30,66 @@ Average of "How much time" questions
   - [Section 10. Learning and Generative Culture](#section-10-learning-and-generative-culture)
   - [Section 11. Transformation and Transformational Leadership](#section-11-transformation-and-transformational-leadership)
 
-### SECTION 1. VERSION CONTROL AND CODE MAINTAINABILITY
+### WHAT IS AN ASSESSMENT TOOL
+
+An Assessment tool is a roadmap that helps to prospect self-improvement, uncover team challenges and opportunities through  capabilities, identifies priorities and tracks the progress.  It is easy to use and produce credible results. The tool provides the ability to review team performance and guide to achieve a desired outcome. It is a tool to drive technology performance improvement in the organization.  The Assessment tool is a survey that will help to measure team performance, understand strength and give recommendations on areas of improvement. It does not collect any data and the responses stay anonymous. The tool will show the areas where the competencies of your team do not reach the level required to reach team goals.  When gaps do not get addressed, they tend to get bigger, eventually causing teams to collapse. Provided recommendations for each section will fill the gaps in your team that allows them to grow and reach required goals. Received score and recommendations can be saved by a user in order to come back and take the survey again. When you compare the competencies defined in the first assessment to the results you received in the second, you will find how your team performance progressed. The process can be repeated until desired outcomes of team performance achieved.  
+
+#### What is the problem and solution  
+
+To manage today business environment challenges organizations need the ability to deliver software with speed and stability. It will help to improve technology and organizational performance.  
+
+#### Who is it for  
+
+IT teams in IITB but any Dev team can benefit from this tool.  
```

Comment:
I'm thinking this sentence should not specifically identify devs or at least expand beyond the devs?

Would **"All teams in IITB looking to continuously improve themselves and support the improvement efforts of their peers"** be somewhat accurate?

Reply:
Changed! I like it better ;) 
Thanks for letting me know about one sentence per line, I didn't know about it;  **"learning git never stops"**